index.html

```diff
@@ -120,27 +120,24 @@ <h1 style="margin-top: 10px">Smart figures with metadata</h1>
         <h1>Install instructions</h1>
         <p>
           <ul>
+            <li>You need to be running OMERO.web 5.0 or later</li>
             <li>
               Download the release from the top of this page, unzip and
               rename the unzipped directory to 'figure'.
             </li>
             <li>
               Place 'figure' <strong>within</strong> a directory that is on your PYTHONPATH, or
-              in the OMERO <code style="display:inline">lib/python/omeroweb/</code> directory. NB: for OMERO 4.4.9 and earlier, you will
-              have to use lib/python/omeroweb/ (other locations not supported).
+              in the OMERO <code style="display:inline">lib/python/omeroweb/</code> directory.
```

Comment:
As we drop OMERO 4.4, should this recommendation be streamlined to put figure in a directory in the PYTHONPATH? Recommending `lib/python/omeroweb` has many practical issues especially when upgrading the server/web.